src/models/MultipleResultsEvent.js

```diff
@@ -73,7 +73,7 @@ export default class MultipleResultsEvent {
 
   isPending = () => Boolean(!!this.pendingTxs && this.pendingTxs.total && this.pendingTxs.total > 0);
 
-  isUpcoming = (address) => this.status === EVENT_STATUS.ORACLE_RESULT_SETTING
+  isUpcoming = (address) => [EVENT_STATUS.PRE_RESULT_SETTING, EVENT_STATUS.ORACLE_RESULT_SETTING].includes(this.status)
```

Comment:
can we rename `isUpcoming` to something more descriptive? is what upcoming?

Reply:
actually, isupcoming is not used now, but we used to have it label oracle result setting event in arbitration tab

Reply:
can we remove it everywhere?

Reply:
removed